assets/css/less/wpbdp.less

```diff
@@ -1,4 +1,5 @@
 @import 'common.less';
+@import 'twenty-twenty-one.less';
```

Comment:
Maybe we can keep this a bit more generic so we can use it for other themes too?

Reply:
It looks well on the other themes, but since twenty twenty one has smaller content area and different styles, it cannot be used on all themes unless we change the css layout of this section

Reply:
File name has been updated :)